pkgs/development/r-modules/generate-r-packages.R

```diff
@@ -2,7 +2,10 @@
 library(data.table)
 library(parallel)
 library(BiocManager)
-cl <- makeCluster(10)
+library(jsonlite)
+
```

Comment:
Could we perhaps put a comment here either explaining how to use the script, or pointing to the documentation?

Reply:
Added a comment directing the user to the manual, though I did not put a link.